docs/How-We-Work/Secrey-Key-Mgmt.md

```diff
@@ -0,0 +1,45 @@
+
+If secret keys are leaked/exposed, this can not only change how the TDP system functions, but this also compromises the security of the data files flowing through the system (and the individuals and families represented in these data). There are two primary avenues that secret key leakage can happen: 
+- a security breach in one or more of the project [tools](https://about.codecov.io/security-update/) that use the secret keys(*relatively speaking, this is more challenging to prevent*), or 
+- a team member with access to the secret keys leaks them (*this is usually unintentional*)
+
+## Mitigation steps
+Below are a list of steps the team is taking to help mitigate the incidence of secret key leakage and stay up-to-date on news that could potentially compromise the security state of our TDP system. This may be updated over time, as additional or alternative solutions are adopted. 
+
+### Current practices
+- Use `git status` terminal command before any commits and pushes to github repo. This should help detect any secret key files that have been modified prior to `git commit`. 
+- Prioritize updating [gitignore](https://git-scm.com/docs/gitignore) before submitting any follow-on PRs that involve use of files that include secret keys. 
+- Secret keys are to be retrieved from cloud.gov for local development purposes. Cloud.gov is a platform that requires these keys, and the dev team has access to the keys stored in the dev environment space. Therefore, this is a more secure approach for retrieving keys than relying on team members to share keys across other platforms/tools.  
+- No production keys will be stored on local machines. Only people who have access to the production space in Cloud.gov will have access to prod keys. 
+- At a minimum, ACF and vendor Tech Leads should be subscribed to tech tools we use to stay up-to-date on news that could impact project security. This includes: CircleCi, CodeCov, [insert others that are relevant]
+- As part of ACF Tech Lead's periodic review of environment variables, secret key rotation will be coordinated.
+
+### Longer-term solutions to be implemented (these are housed under epic #972 ):
+- Automated tool will be added to detect secret keys before changes are committed project repo (#965) 
+- Add .dockerconfig file as an extra layer of security against accidentally leaking secrets (#544)
+- Automated testing step will be added in CircleCI to check for secret keys (#966)
+- (#967)
+- (#968)
+- (#969)
+
+## Communication protocol if secret keys are leaked
+Any member of the TDP who notices secret key leakage should alert the full TDP project team immediately as follows:
+
+- Send alert via Microsoft Teams `General` channel and be sure to cc: Product Owner, ACF Tech Lead, Product Manager, vendor Tech Lead. 
```

Comment:
this will need to be revisited if TDP primary chat communication tool is changed

Reply:
:crossed_fingers:

Reply:
This was resolved, for now

Reply:
But maybe we shuold have backup. If Teams isn't available, send via email and Mattermost.